plugin.video.mlbtv/resources/lib/account.py

```diff
@@ -248,6 +248,14 @@ def get_stream(self, content_id):
         return stream_url, headers
 
     def get_stream_quality(self, stream_url):
+        #Check if inputstream adaptive is on, if so warn user and return master m3u8
+        if xbmc.getCondVisibility('System.HasAddon(inputstream.adaptive)'):
```

Comment:
Unless the wiki has the wrong information regarding this boolean condition, I would suggest to use the json api instead and check for the *enable* state of the addon. This way you will trigger this message on every kodi install since `inputstream.adaptive` is bundled with kodi by default. JSON call:
```
{"jsonrpc":"2.0","method":"Addons.GetAddonDetails","params":{"addonid":"inputstream.adaptive","properties":["enabled"]},"id":4}
```

Reply:
The if statement does return false when inputstream is installed but not enabled. Same if is currently used in PS Vue addon. Not saying it's proper but it does work as expected.

Reply:
Nice to know, we need to update the wiki then :)